runelite-mixins/src/main/java/net/runelite/mixins/RSClientMixin.java

```diff
@@ -446,15 +446,47 @@ public boolean isClanMember(String name)
 	@Inject
 	public static void experiencedChanged(int idx)
 	{
-		ExperienceChanged experienceChanged = new ExperienceChanged();
+		SkillUpdate update = new SkillUpdate();
```

Comment:
Why not split all these events to 3 different ones? SkillExperiencesChanged, RealSkillLevelsChanged, BoostedSkillLevelsChanged?